File: app/src/main/java/co/tpcreative/supersafe/model/Categories.java
package co.tpcreative.supersafe.model;

import java.io.Serializable;

public class Categories implements Serializable {
    public int id;
    public String name;


    public Categories(){
        this.id = 0;
        this.name = "";
    }

    public Categories(int id,String name){
        this.id = id;
        this.name = name;
    }

}

File: app/src/main/java/co/tpcreative/supersafe/model/Categories.kt
package co.tpcreative.supersafe.model
import java.io.Serializable

class Categories : Serializable {
    var id: Int
    var name: String?

    constructor() {
        id = 0
        name = ""
    }
    constructor(id: Int, name: String?) {
        this.id = id
        this.name = name
    }
}